### PR TITLE
EZP-29761: RichText Editor crashes after adding link to the content into unordered list

### PR DESCRIPTION
**Pull request by dew326**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-29761
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/js/alloyeditor/src/toolbars/config/ez-embed.js [hunk 1]
```diff
@@ -5,15 +5,7 @@ export default class EzEmbedConfig extends EzConfigBase {
...
-        this.buttons = [
-            'ezmoveup',
-            'ezmovedown',
-            'ezembedupdate',
-            'ezembedleft',
-            'ezembedcenter',
-            'ezembedright',
-            'ezblockremove',
-        ];
+        this.buttons = ['ezmoveup', 'ezmovedown', 'ezembedupdate', 'ezembedleft', 'ezembedcenter', 'ezembedright', 'ezblockremove'];
```
[the remark this answers is not in the document]
Comment (micszo) [reply src/bundle/Resources/public/js/alloyeditor/src/toolbars/config/ez-embed.js]: https://media.giphy.com/media/tFK8urY6XHj2w/giphy.gif

**Comment by mateuszbieniek** [comment]
@micszo for the last couple of times it ended with micro-release/release of a package with a fix and I asked the customer to `composer update`.

**Comment by micszo** [comment]
OK, then I suppose we're good to go. 🙃

**Comment by mateuszbieniek** [comment]
Then I'm pinging @lserwatka :)

**Comment by lserwatka** [comment]
@dew326 you can merge it up

**Pull request merged**